### DISCUSSION TITLE
Where can I place resources for tests and running?

### DISCUSSION THREAD

**Discussion by RalfKow** [python-poetry/poetry#6584] [Q&A]
How do I add nessessary resources for tests and src files? I'm used to a file resources in the folder src and tests to manage all nessessary resources f.E. an example json file

**Comment by neersighted** [comment]
This isn't a Poetry specific question, but instead a Python packaging question. You'll want to consult `importlib.resources` for the most robust and portable way to access resource/data files from a Python module/package:

* https://docs.python.org/3.11/library/importlib.resources.html
* https://importlib-resources.readthedocs.io/en/latest/using.html
